core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala

```diff
@@ -69,96 +69,25 @@ class LocalEntityProvider(requestArguments: EntityRequestArguments,
-        } else {
-          // system and request both have cache enabled. Check for existence and staleness of cache
-          cacheStaleness(dataAccess, localContext).flatMap {
-            case None =>
-              // cache does not exist - return uncached
-              logger.info(
-                s"entity statistics cache: miss (cache does not exist) [${workspaceContext.workspaceIdAsUUID}]"
-              )
-              calculateMetadataResponse(dataAccess, countsFromCache = false, attributesFromCache = false, localContext)
-            case Some(0) =>
-              // cache is up to date - return cached
-              logger.info(s"entity statistics cache: hit [${workspaceContext.workspaceIdAsUUID}]")
-              calculateMetadataResponse(dataAccess, countsFromCache = true, attributesFromCache = true, localContext)
-            case Some(stalenessSeconds) =>
-              // cache exists, but is out of date - check if this workspace has any always-cache feature flags set
-              cacheFeatureFlags(dataAccess, localContext).flatMap { flags =>
-                if (flags.alwaysCacheTypeCounts || flags.alwaysCacheAttributes) {
-                  setTraceSpanAttribute(localContext,
-                                        AttributeKey.booleanKey("alwaysCacheTypeCountsFeatureFlag"),
-                                        java.lang.Boolean.valueOf(flags.alwaysCacheTypeCounts)
-                  )
-                  setTraceSpanAttribute(localContext,
-                                        AttributeKey.booleanKey("alwaysCacheAttributesFeatureFlag"),
-                                        java.lang.Boolean.valueOf(flags.alwaysCacheAttributes)
-                  )
-                  logger.info(
-                    s"entity statistics cache: partial hit (alwaysCacheTypeCounts=${flags.alwaysCacheTypeCounts}, alwaysCacheAttributes=${flags.alwaysCacheAttributes}, staleness=$stalenessSeconds) [${workspaceContext.workspaceIdAsUUID}]"
-                  )
-                } else {
-                  logger.info(
-                    s"entity statistics cache: miss (cache is out of date, staleness=$stalenessSeconds) [${workspaceContext.workspaceIdAsUUID}]"
-                  )
-                  // and opportunistically save
-                }
-                calculateMetadataResponse(dataAccess,
-                                          countsFromCache = flags.alwaysCacheTypeCounts,
-                                          attributesFromCache = flags.alwaysCacheAttributes,
-                                          localContext
-                )
-              } // end feature-flags lookup
-          } // end staleness lookup
-        } // end if useCache/cacheEnabled check
-      } // end transaction
-    } // end root trace
+  ): Future[Map[String, EntityTypeMetadata]] = Future.successful(Map.empty[String, EntityTypeMetadata])
```

Comment:
LocalEntityProvider's `entityTypeMetadata()` method is now non-functional. But, it should never be called, so it's ok that it is non-functional. We will be deleting this whole provider soon, so I'm not worried about keeping this code clean.